src/routes/components/components.json

```diff
@@ -853,6 +853,16 @@
 		"title": "svelte-material-ui",
 		"url": "https://github.com/hperrin/svelte-material-ui"
 	},
+	{
+		"addedOn": "2020-09-29T14:39:13Z",
+		"category": "Design System",
+		"description": "A Svelte.js based UnoCSS UI library that allows you to make websites",
+		"npm": "@ikun-ui/core",
+		"stars": 650,
```

Comment:
This line is not necessary anymore.
Only `src/lib/stars.json` is used to display the stars count
```suggestion
```